nsxt/resource_nsxt_policy_ipsec_vpn_service.go

```diff
@@ -96,6 +98,28 @@ func updateNsxtPolicyIPSecVpnService(connector *client.RestConnector, gwID strin
 	return err
 }
 
+func resourceNsxtPolicyIPSecVpnServiceImport(d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
+	importID := d.Id()
+	s := strings.Split(importID, "/")
```

Comment:
Lets use full service path for consistency. In future, we plan to support policy path in all imports

Reply:
done